api/routes.go

```diff
@@ -38,9 +38,7 @@ func (s *server) routes() {
 	api.HandleFunc("/{account}/movers/{group}/{name}", s.MoverDeleteHandler).Methods(http.MethodDelete)
 	// api.HandleFunc("/{account}/movers/{group}/{name}/{id}", s.MoverUpdateHandler).Methods(http.MethodPut)
 
-	api.HandleFunc("/{account}/movers/{group}/{name}/runs", s.StartTaskHandler).Methods(http.MethodPost)
-	api.HandleFunc("/{account}/movers/{group}/{name}/runs", s.StopTaskHandler).Methods(http.MethodPut)
-	api.HandleFunc("/{account}/movers/{group}/{name}/startStop", s.StartStopTaskHandler).Methods(http.MethodPut)
+	api.HandleFunc("/{account}/movers/{group}/{name}/runs", s.MoverUpdateHandler).Methods(http.MethodPut)
```

Comment:
The route should be `/{account}/movers/{group}/{name}/{id}` (basically what's commented out above)

Reply:
As discussed earlier, In stopping & starting a task there is no use of  id, because, at a time only one task can run. I am using CurrentTaskExecutionARN to stop the task.